src/components/Editor/Editor.jsx

```diff
@@ -152,6 +169,8 @@ class Editor extends React.Component {
     let cursor = cursorPosition;
     const selection = editor.getDoc().getSelection();
 
+    this.setState({ keyMap: this.keyMap() });
```

Comment:
Isn't it enough to update `this.state.keyMap` on focus only? As when we use `Configurator`, we will need to focus the editor again to keep coding? If not, please let me know, I am simply asking away. :)

Reply:
Yes, good catch! My bad, this was my initial placement before I found `focus`, and it didn't quite work, but looks like I forgot to remove it when I was finished.